security/oc_tls.h

```diff
@@ -155,16 +155,78 @@ void oc_tls_use_pin_obt_psk_identity(void);
 int oc_tls_pbkdf2(const unsigned char *pin, size_t pin_len, oc_uuid_t *uuid,
                   unsigned int c, uint8_t *key, uint32_t key_len);
 
-/* Internal interface for refreshing identity certficate chains */
-void oc_tls_refresh_identity_certs(void);
-void oc_tls_remove_identity_cert(oc_sec_cred_t *cred);
+/**
+ * @brief Internal interface for adding new identity certificate chains.
+ *
+ * Iterate over all credentials, check if a credential is associated with a leaf
+ * identity certificate. If the identity certificate doesn't exist in the global
+ * list of identity certificates then create a new identity certificate item and
+ * add it to the list.
+ */
+void oc_tls_add_new_identity_certs(void);
```

Comment:
`oc_tls_add_new_identity_certs` -> `oc_tls_resolve_identity_certs` ?

Reply:
sure that works too